tests/src/testContext/runner-test-context.ts

```diff
@@ -0,0 +1,8 @@
+import type { TestContext } from 'vitest';
+import type { PodmanDesktopRunner } from '../runner/podman-desktop-runner';
+
+declare module 'vitest' {
+  export interface PDRunnerTestContext extends TestContext {
```

Comment:
```suggestion
  export interface PodmanDesktopRunnerTestContext extends TestContext {
```

(or just RunnerTestContext)

Reply:
RunnerTestContext sounds good.